projects/common-consumption/src/lib/content-details/player-toc/player-toc.component.scss

```diff
@@ -0,0 +1,270 @@
+
+
+/* toc css */
+.sbtoc-container {
+  
+    .sbselectpopover {
+      height:calc(var(--base-block-space) * 8);
+      padding:var(--base-block-space) calc(var(--base-block-space) * 2);
+      display:flex;
+      align-items: center;
+      justify-content: space-between;
+      cursor: pointer;box-shadow: 0 4px 7px 0 rgba(0,0,0,0.1);
+      &--bg-white {
+        background:var(--white);
+      }
+      h4 {
+        font-size:1em;
+        color:var(--primary-400);margin: 0;
+        padding: 0px;
+        font-weight:bold;
+        width:100%;
+      }
+    }
+    
+    .sbtoc-filter {
+      height:calc(var(--base-block-space) * 8);
+      padding:var(--base-block-space) calc(var(--base-block-space) * 2);
+      box-shadow: 0px 2px 10px -5px rgba(0, 0, 0, 0.25);display: inline-flex;
+      align-items: center;
+      width:100%;overflow: hidden;
+      overflow-x: auto;
+      border-bottom:1px solid #e0e0e0;
+      // &::-webkit-scrollbar {
+      //   height:0px;
+      // }
+      &--bg-gray{
+      background:var(--gray-0);
+      }
+      &__grid {
+        display: inline-flex;
+        align-items: center;
+        margin: 0px;

```

Comment:
All Margins & paddings better to call it using mixins to avoid rework on rtl support